core/ir/riscv64/instr_create_api.h.in

```diff
@@ -361,13 +361,13 @@
 
 /** @name No-operand instructions */
 /** @{ */ /* doxygen start group; w/ DISTRIBUTE_GROUP_DOC=YES, one comment suffices. */
+
 /**
  * This INSTR_CREATE_xxx macro creates an instr_t with opcode OP_xxx, automatically
  * supplying any implicit operands.
  * \param dc The void * dcontext used to allocate memory for the instr_t.
  */
-#define INSTR_CREATE_ecall(dc) instr_create_0dst_0src(dc, OP_ecall)
-#define INSTR_CREATE_nop(dc) instr_create_0dst_0src(dc, OP_hint)
+@INSTR_MACROS@
```

Comment:
Add comment that this is auto-generated by codec.py. Same for @OPCODES@.

Reply:
Done